databallpy/load_data/tracking_data/_quality_check_tracking_data.py

```diff
@@ -0,0 +1,184 @@
+import warnings
+
+import numpy as np
+import pandas as pd
+
+from databallpy.features.velocity import get_velocity
+from databallpy.warnings import DataBallPyWarning
+
+
+def _quality_check_tracking_data(
+    tracking_data: pd.DataFrame, framerate: float, periods: pd.DataFrame
+) -> None:
+    """Function that does a quality check on the tracking data
+
+    Args:
+        tracking_data (pd.DataFrame): tracking data
+        framerate (float): framerate of the tracking data
+        periods (pd.DataFrame): holds information on start and end of all periods
```

Comment:
Add `"Raises"` to doccstring, it should tell all `databallpy` warnings that are possibly raised when calling this function and at what condition.